enterprise/migrations/0137_enrollment_email_update.py

```diff
@@ -0,0 +1,230 @@
+from logging import getLogger
+from django.db import migrations
+
+LOGGER = getLogger(__name__)
+
+def update_email(apps, schema_editor):
```

Comment:
why not just do this in Django Admin?
IE we shouldn't need to be updating templates anyways...
I get for initial population, but we're adjusting an already existing one. No need to deal with migration headaches for that.

admin/enterprise/enrollmentnotificationemailtemplate/ is the path by the way. ie (`http://localhost:18000/admin/enterprise/enrollmentnotificationemailtemplate/` or courses-internal.stage.edx.org/admin/enterprise/enrollmentnotificationemailtemplate/)

Reply:
(This shouldn't really be a task for developers) given we actually expose these templates

Reply:
This makes sense (and how I was doing it initially when testing), but I figured for it to be updated on everyone's local it would be better to make a migration and tests? But I can also scratch this entirely and just post the html into stage and prod

Reply:
I would say any further updates to templates can be done in Django admin. Not as nice as having a git history but there is Django history too I think